pyproject.toml

```diff
@@ -1,11 +1,11 @@
 [tool.codespell]
 check-filenames = true
 check-hidden = true
-ignore-words-list = "ot"
+ignore-words-list = "ot,gallary"
```

Comment:
Just wondering, why is `gallary` not a typo?

Reply:
It was a third-party URL, hence didn't want to disturb it

https://co-founder.ai?utm_source=llama-index&utm_medium=gallary&utm_campaign=alpha